client/src/components/AddUserForm.jsx

```diff
@@ -27,9 +29,8 @@ const AddUserForm = ({ hideForm }) => {
             <label className="heading user-add__label" htmlFor="username">
               User_Name
             </label>
```

Comment:
`TextInput` will already create a label as it creates an entire `fieldset` which includes an input and corresponding label element. Instead of using this label externally, can we just pass `label="user_name"` to the `TextInput`?